android/libraries/rib-base/src/main/kotlin/com/uber/rib/core/Worker.kt

```diff
@@ -56,4 +56,17 @@ public interface Worker {
 
   /** Called when the worker is stopped. */
   @JvmDefault public fun onStop() {}
+
+  public companion object {
+
+    /**
+     * Returns the CoroutineContext defined at [Worker].
+     *
+     * In testing environments that rely on mockito, a Mocked [Worker] will return a null
+     * coroutineContext.
+     */
+    public fun Worker.getBindingContext(): CoroutineContext {
```

Comment:
I prefer it private, and inside `WorkerBinder` file at top level. Why do we need to expose this ext fn?

Reply:
oh that's right, we discussed internally. Let me update in a bit